cms/tests/test_plugins.py

```diff
@@ -280,47 +280,6 @@ def test_plugin_breadcrumbs(self):
         self.assertTrue('/edit-plugin/%s/'% column.pk, text_breadcrumbs[1]['url'])
         self.assertTrue('/edit-plugin/%s/'% text_plugin.pk, text_breadcrumbs[2]['url'])
 
-    def test_add_cancel_plugin(self):
```

Comment:
don't think so.. it's very straightforward so should be fine with testing the new behavior